pom.xml

```diff
@@ -93,15 +95,6 @@
       <properties>
         <hadoop.version>${hadoop2.version}</hadoop.version>
       </properties>
-      <dependencies>
-        <!-- TestHBOSSContractDistCp hard-codes some HDFS classes in Hadoop 2 -->
-        <dependency>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>hadoop-hdfs</artifactId>
-          <version>${hadoop.version}</version>
-          <scope>test</scope>
-        </dependency>
-      </dependencies>
     </profile>
```

Comment:
Moved to general test dependencies on hboss pom module, as it was then required for both hadoop profiles (see comment above).